packages/csv-parser-price/src/main.js

```diff
@@ -55,13 +56,14 @@ export default class CsvParserPrice {
 
     highland(input)
       // Parse CSV return each row as object
+      .through(stripBom())
       .through(
         csv({
           separator: this.delimiter,
           strict: true,
         })
       )
-      .map((a) => {
+      .map(a => {
```

Comment:
hmm, these formatting changes shouldn't happen. Could you please run `yarn format` in the root package folder to undo these changes? and maybe `yarn install` if you're running some old dependencies locally?

Reply:
It happens automatically. Sure will update.

Reply:
done